### PR TITLE
add `<Icon type="status-icon" status={success,error,warning} />`

### PR DESCRIPTION
**Pull request by stephenmathieson**
@anthonyshort @dominicbarnes @stevenmiller888

### CODE REVIEW

File: lib/icons/success.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (anthonyshort) [inline lib/icons/success.js]: OCD: Extra space :D

File: lib/icons/error.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+import element from 'dekujs/virtual-element';
+
+export default function render(color) {
+  return (
+    <svg width="16px" height="16px" viewBox="0 0 16 16">
+      <title>error</title>
```
Comment (anthonyshort) [inline lib/icons/error.js]: You can actually remove the title, desc, defs elements from all of these

File: lib/icons/success.js (hunk 1 above)
Comment (stephenmathieson) [inline lib/icons/success.js]: makes editing in vim significantly easier ;)

File: lib/icons/error.js (hunk 2 above)
Comment (stephenmathieson) [inline lib/icons/error.js]: cool, will do

**Comment by anthonyshort** [comment]
https://segment.atlassian.net/browse/WARE-54

I've attached some updated icons in jira. There were some extra elements and groups in the ones I originally sent. They will probably mess up Firefox and Safari if we don't update them. 

Other than that, it's all good to me.

**Comment by stephenmathieson** [comment]
thought about abstracting this a bit further:

``` js
import StatusIcon from 'segmentio/icons';

// ...
<StatusIcon type="success|warning|error" />
```

thoughts?

**Comment by anthonyshort** [comment]
Yeah I'd be down for getting these 3 icons to be a separate thing. They're used in the alert boxes too (but bigger). Since they're not the same style as the other icons, it could be good to have them separate instead.

So +1!

**Comment by stephenmathieson** [comment]
~~cool, i'll update for `StatusIcon`~~

done

**Comment by stevenmiller888** [comment]
LGTM

**Pull request merged**